pkg/bench/get.go

```diff
@@ -47,6 +51,56 @@ type Get struct {
 // Prepare will create an empty bucket or delete any content already there
 // and upload a number of objects.
 func (g *Get) Prepare(ctx context.Context) error {
+
+	/*
+	 * Prepare the bench by listing object from the bucket
+	 */
+	if g.GetFromListing {
```

Comment:
We do not use block comments:
```suggestion
	// Prepare the bench by listing object from the bucket
	if g.GetFromListing {
```

Reply:
fixed